global-setup.e2e.ts

```diff
@@ -7,9 +7,9 @@ import stopLocalRegistry from './tools/scripts/stop-local-registry';
 export async function setup() {
   await globalSetup();
   await startLocalRegistry();
-  execSync('npm install -D @code-pushup/cli@e2e');
-  execSync('npm install -D @code-pushup/eslint-plugin@e2e');
-  execSync('npm install -D @code-pushup/coverage-plugin@e2e');
+  execSync('npm install -D @code-pushup/cli@e2e --force');
+  execSync('npm install -D @code-pushup/eslint-plugin@e2e --force');
+  execSync('npm install -D @code-pushup/coverage-plugin@e2e --force');
```

Comment:
`force` option was necessary due to the move to the major version update of Nx

Reply:
Is it because of `@nx/devkit` peer dependency? We should probably expand `^17.0.0` to `^17.0.0 || ^18.0.0`, so that we're compatible with Nx 18.